src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Square extends FigureForOther {
```

Comment:
rewrite code accordingly
```suggestion
public class Square extends Figure {
```